ci-operator/step-registry/gather/must-gather/gather-must-gather-commands.sh

```diff
@@ -10,6 +10,12 @@ then
 	exit 0
 fi
 
+if test -f "${SHARED_DIR}/proxy-conf.sh"
```

Comment:
add a comment here about what this is doing - we may want to actually make this a feature of step registry if necessary and i'd like the comment to clearly indicate what it's doing

Reply:
Done